packages/happo-target-react-native/src/initializeDriver.js

```diff
@@ -0,0 +1,99 @@
+const wd = require('wd');
+const childProcess = require('child_process');
+
+const startAppium = ({
+  appiumPath,
+  appiumCwd,
+  appiumEnv,
+  appiumServerPort,
+}) => (
+  new Promise((resolve) => {
+    const args = [
+      '-p',
+      String(appiumServerPort),
+    ];
+    const appiumProc = childProcess.spawn(appiumPath, args, { cwd: appiumCwd, env: appiumEnv });
+
+    let loadedAppium = null;
+    appiumProc.stdout.on('data', (data) => {
+      if (loadedAppium) return;
+      console.log(`[Appium]: ${data}`);
+      if (data.indexOf('Appium REST http interface listener started') >= 0) {
+        loadedAppium = true;
+        resolve();
+      }
+    });
+    appiumProc.stderr.on('data', (data) => {
+      console.log(`[Appium]: Error ${data}`);
+      appiumProc.kill();
+    });
+    process.on('error', (e) => {
+      console.log('error', e);
+    });
+    process.on('exit', (e) => {
+      console.log(e);
+      console.log('[Appium]: Exiting...');
+      appiumProc.kill();
+    });
+  })
+);
+
+let driver;
```

Comment:
Do you need to reassign this? It looks like you only use it to resolve a promise, so maybe it can be inline in the method?